src/App/Fossa/Config/ConfigFile.hs

```diff
@@ -198,6 +198,7 @@ data ConfigFile = ConfigFile
   , configTelemetry :: Maybe ConfigTelemetry
   , configCustomLicenseSearch :: Maybe [ConfigGrepEntry]
   , configKeywordSearch :: Maybe [ConfigGrepEntry]
+  , configIgnoreOrgWideCustomLicenseScanConfigs :: Bool
```

Comment:
[Optional] I would recommend something like `orgWideCustomLicenseScanConfigPolicy` with an enum like 
```haskell
data OrgWideCustomLicenseConfigPolicy = Use | Ignore
```

It's a bit easier to see what's going on and also leaves open the possibility for more than two policies.